concepts-wip/bools/about.md

```diff
@@ -0,0 +1,126 @@
+# Bools
+
+Crystal has a type known as [`Bool`][bools], it is used to represent the values `true` and `false`.
+You can use logical operators (`!`, `||`, `&&`) which can be used to combine Bools and create more complex expressions:
```

Comment:
```suggestion
```
You're just about to introduce these. I don't think the repetition is needed. I suggest deleting.
(same in introduction)